update-master.yml

```diff
@@ -0,0 +1,18 @@
+- hosts: server
+
+
+# https://github.com/jnv/ansible-role-unattended-upgrades    
+    
+  roles:
+  - role: jnv.unattended-upgrades
+    unattended_origins_patterns:
+    - 'origin=Ubuntu,archive=${distro_codename}-security'
+    - 'o=Ubuntu,a=${distro_codename}-updates'
+    unattended_package_blacklist: [playonlinux, wine]
+    unattended_mail: "{{adfc_notification_email}}"
+    unattended_install_on_shutdown: false
+    unattended_verbose: 0
+    unattended_random_sleep: 3600
+    unattended_automatic_reboot: false
+#    unattended_automatic_reboot_time: 03:00
+    unattended_update_days: {"Sat"}
```

Comment:
Das verstehe ich nicht. Willst du den ucs-master damit updaten? Oder was ist das für ein server?

Reply:
UCS hat ein ganz anderen Vorgang für Updates. Außerdem muss bei einem Reboot das Passwort eingegeben werden.

Reply:
hosts: server bezieht sich auf das inventory.yml
Somit würde auch der UCS-Master das unattended upgrade erhalten.

Eventuell müsste man diesen in eine separate Untergruppe verschieben. (z.b. hosts: server_manual_updates)

Reply:
Oder eine neue Gruppe "unattended_update" anlegen?

Reply:
Welche Server haben wir denn außer dem ucs-master? Proxmox?

Reply:
Wollen wir die Datei löschen und den rest mergen?

Reply:
ok, machen wir so. 
Automatisierte Updates auf den Serversystemen sind nicht wirklich eine gute Idee.